hack/build_cli.sh

```diff
@@ -14,10 +14,10 @@ do
     if [[ $os == "windows" ]]; then
```

Comment:
It won't let me comment on the exact line, but I think we need ARM as another `arch` on line 4 for Darwin specifically. maybe other OSes too? not 100% sure but I know Darwin needs ARM 64